src/components/Card.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import styles from "./Card.module.css";
+import Star from "./Star";
+import defaultCardImg from "/src/assets/default-background.png";
+
+function Card({ cardData }) {
+  const calculatePassedTime = (time) => {
+    const MINUTE = 60 * 1000;
+    const HOUR = 60 * MINUTE;
+    const DAY = 24 * HOUR;
+    const MONTH = 31 * DAY;
+    const YEAR = 12 * MONTH;
+
+    const currentTime = new Date();
+    const createdTime = new Date(time);
+    const timeDiff = currentTime - createdTime;
+
+    switch (true) {
+      case timeDiff < 2 * MINUTE:
+        return "1 minute ago";
+      case timeDiff < HOUR:
+        return `${Math.floor(timeDiff / MINUTE)} minutes ago`;
+      case timeDiff < 2 * HOUR:
+        return "1 hour ago";
+      case timeDiff < DAY:
+        return `${Math.floor(timeDiff / HOUR)} hours ago`;
+      case timeDiff < 2 * DAY:
+        return "1 day ago";
+      case timeDiff < MONTH:
+        return `${Math.floor(timeDiff / DAY)} days ago`;
+      case timeDiff < 2 * MONTH:
+        return "1 month ago";
+      case timeDiff < YEAR:
+        return `${Math.floor(timeDiff / MONTH)} months ago`;
+      case timeDiff < 2 * YEAR:
+        return "1 year ago";
+      default:
+        return `${Math.floor(timeDiff / YEAR)} years ago`;
+    }
+  };
+
+  const changeDateFormat = (time) => {
+    return Intl.DateTimeFormat("kr").format(new Date(time));
+  };
+
+  return (
+    cardData && (
```

Comment:
```suggestion
  if(!cardData) return null
  return (
    ...
 )
```
(삭제할 코드지만) 이 코드가 좀 더 일반적인 것 같아요.